fsspec/implementations/tests/test_sftp.py

```diff
@@ -221,10 +221,10 @@ def test_mkdir_create_parent(ssh, path):
 def test_makedirs_exist_ok(ssh, path):
     f = fsspec.get_filesystem_class("sftp")(**ssh)
 
-    f.makedirs(path)
+    f.makedirs(path, exist_ok=False)
 
     with pytest.raises(FileExistsError, match=path):
-        f.makedirs(path, exist_ok=False)
+        f.makedirs(path)
```

Comment:
Probably better to keep the original line here, since that's explicitly what this function is testing.

Reply:
Sure, line 224 can stay the same right?

Reply:
Yes, although explicit in every call is fine (in case we ever change the default in makedirs).